src/js/components/Anchor/__tests__/__snapshots__/Anchor-test.tsx.snap

```diff
@@ -899,6 +899,22 @@ exports[`Anchor renders size specific theming 1`] = `
         Anchor with icon
       </span>
     </a>
+    <a
+      class="c8"
+    >
+      <span
+        class="c17"
+        style="display: inline-flex;"
+      >
+        <svg
+          color="#7D4CDB"
```

Comment:
So, `color` isn't really an `<svg />` attribute. StyledIcon.js converts the `color` property to `stroke` and `fill` in CSS. I'm wondering if we should use an actual grommet-icon to go full circle and make the snapshot looks more real.

Reply:
I can do that, will update this test with that.

Reply:
Done -- you'll see class `c20` applies the appropriate color to the icon.